CONTRIBUTING.md

```diff
@@ -0,0 +1,289 @@
+Hello! Thank you for choosing to help contribute to one of the SendGrid open source libraries. There are many ways you can contribute and help is always welcome. We simply ask that you follow the following contribution policies.
```

Comment:
This contributing file does not apply to this repo. It's specific to our docs repo. Please remove it from this PR. Thanks!